python/cudf/cudf/pylibcudf_tests/test_column_factories.py

```diff
@@ -0,0 +1,266 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import pyarrow as pa
+import pytest
+from utils import DEFAULT_STRUCT_TESTING_TYPE, assert_column_eq
+
+from cudf._lib import pylibcudf as plc
+
+size = 3
```

Comment:
This is a very generically named constant, and it's not used all that much. Do we benefit a lot from it? Could we use a fixture so that it's more obvious which tests are using it? Can we rename it to be a bit more informative?